### ISSUE TITLE
SPM Support

### ISSUE THREAD

**Issue by iowais-dev** [razorpay/razorpay-pod#178]
We need swift package manager support so we can avoid integration errors and save our precious time.

**Comment by jonathanrajya** [comment]
I have ported it to use spm, you can find it here: [Razorpay SPM](https://github.com/jonathanjr3/Razorpay-spm) use main branch for the update rule as I haven't made any release.

**Comment by kmrinmoy07** [comment]
Yes please look into this, this is an important requirement for us as well

**Comment by salvatoreboemia** [comment]
News?
